src/arrayMethodSort.js

```diff
@@ -4,8 +4,23 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 0; j < this.length - 1; j++) {
```

Comment:
The inner loop can iterate until 'this.length - i - 1' instead of 'this.length - 1' since the last 'i' elements are already sorted in previous iterations.